README.md

```diff
@@ -27,7 +27,39 @@ Notebooks uses AI Platform version 1.14, we had to code in Python 3.5 (no
 f-strings or type hinting)
 
 ### Jupyter via remote execution into AI Platform Notebook Instance
-Coming Soon - Some documentation already complete by team
+Prerequisites: 
+1. You must have Anaconda/Jupyter Notebook installed 
+2. If using a windows machine, you must have PuTTY installed
+3. You must have Cloud SDK installed: https://cloud.google.com/sdk/install
+
+Helpful Links: 
+https://cloud.google.com/ai-platform/notebooks/docs/ssh-access
+
+Instructions:
+1. On the AI Platform Notebook Instance you created,
+open your VM Instance details.
+On the dropdown for Remote access, select "view gcloud command"
+Example: 
+gcloud compute --project "project-name" ssh --zone "us-east1-d" "instance-name"
```

Comment:
I think putting this a code bloc would be helpful. For example:
```
gcloud compute --project <project-name> ssh --zone <zone> 
```
Does the command require the quotes? If not, I personally find using the <> to be clearer that this section is where you put your own inputs and quotes are not needed (I think this is also consistent with google documentation).

Reply:
Good call out! Yes, the command requires the quotes and the zone. I'll call that out.

Reply:
Interesting that the quotes are needed given that for the gcloud ai platform jobs command they aren't! There might be something I'm unaware of there. I'm also curious about the required zone when the instance is already created. I wonder if you can change the zone for an existing notebook this way, or if the instance was created in one zone and you specify a different one here if you would get an error.